README.md

```diff
@@ -71,7 +71,7 @@ This code would make most sense as a...
 - ...helper method.
 
 ```text
-Your answer...
+Helper Method. I'd choose this method because  
 ```
```

Comment:
:+1:  a Helper method would make the most sense because these are intended to "help" render HTML based on conditions. Controller methods concern how data moves between routes. Model mehtods concern how data goes to and comes from the database.